pennylane/tracker.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
Should the name of this file be `device_tracker.py`?

Reply:
It was.  I could make it `device_tracker.py` again.